docs/README.md

```diff
@@ -67,6 +67,26 @@ $ export IONOSCLOUD_PASSWORD='password'
 $ rspec spec
 ```
 
+### Getting input from a file
+
+In order to make passing JSON or array arguments easier, we allow you to specify a file which holds the arguments for a command. This is done with the
+`--extra-config EXTRA_CONFIG_FILE_PATH` or `-e EXTRA_CONFIG_FILE_PATH`. The file is expected to contain a JSON which should map the name of arguments to their expected values. Here is an example of such a file.
+
+```json
+{
+  "ionoscloud_username": "username",
+  "ionoscloud_password": "password",
+  "name": "test_datacenter_name",
+  "location": "de/txl"
+}
+```
+
+When using the command `knife ionoscloud datacenter create --extra-config EXTRA_CONFIG_FILE_PATH` where EXTRA_CONFIG_FILE_PATH is the name of the file containing the JSON above, the arguments described in the file will be used. 
```

Comment:
Missed to update this to: `EXTRA_CONFIG_FILE_PATH is the absolute path of the file ...`.